flink-runtime/src/main/java/org/apache/flink/runtime/operators/chaining/ChainedDriver.java

```diff
@@ -64,12 +64,12 @@
 
 	
 	public void setup(TaskConfig config, String taskName, Collector<OT> outputCollector,
-					  AbstractInvokable parent, UserCodeClassLoader userCodeClassLoader, ExecutionConfig executionConfig,
-					  Map<String, Accumulator<?,?>> accumulatorMap)
+			AbstractInvokable parent, UserCodeClassLoader userCodeClassLoader, ExecutionConfig executionConfig,
```

Comment:
`UserCodeClassLoader userCodeClassLoader` -> `ClassLoader userCodeClassLoader`
also its callers like `BatchTask::initOutputs` and so on up the call stack (`DataSourceTask::initOutputs`)

Reply:
We need to pass `UserCodeClassLoader` into `ChainedDriver.setup()` so that we can forward it to `DistributedRuntimeUDFContext` where `UserCodeClassLoader.registerReleaseHookIfAbsent()` will be called.

Reply:
right, sorry I overlooked that